pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -130,13 +132,17 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 				{
 					APIGroups: []string{coordinationv1.GroupName},
 					Resources: []string{"leases"},
-					Verbs:     []string{"create"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
 				},
 				{
-					APIGroups:     []string{coordinationv1.GroupName},
-					Resources:     []string{"leases"},
-					ResourceNames: []string{"druid-leader-election"},
-					Verbs:         []string{"get", "update", "patch"},
+					APIGroups: []string{batchv1.GroupName},
+					Resources: []string{"jobs"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
+				},
+				{
+					APIGroups: []string{batchv1beta1.GroupName},
+					Resources: []string{"cronjobs"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
```

Comment:
```suggestion
					Verbs:     []string{"get", "list", "watch", "delete"},
```
Should be sufficient to clean up the `CronJob`s, right?

Reply:
Right. Done